platform-dom/src/main/java/com/softicar/platform/dom/element/IDomElement.java

```diff
@@ -32,6 +33,20 @@ public interface IDomElement extends IDomNode {
 	 */
 	DomElementTag getTag();
 
+	/**
+	 * Convenience method to append this {@link IDomElement} to the given
+	 * {@link IDomParentElement}.
+	 *
+	 * @param parent
+	 *            the {@link IDomParentElement} (never <i>null</i>)
+	 * @return this
+	 */
+	default IDomElement appendTo(IDomParentElement parent) {
```

Comment:
How did we survive the last 16 years without that method?

Reply:
That is beyond me. :D